doc/user-guide/dask.rst

```diff
@@ -2,195 +2,151 @@
-and then chunking the data using the ``chunk`` method, e.g.,
-``xr.open_dataset('example-data.nc').chunk({'time': 10})``.
+    See :ref:`io.zarr` for more details.
 
-To open multiple files simultaneously in parallel using Dask delayed,
-use :py:func:`~xarray.open_mfdataset`::
+.. tab:: NetCDF
 
-    xr.open_mfdataset('my/files/*.nc', parallel=True)
+    Open a single netCDF file with :py:func:`~xarray.open_dataset` and supplying a ``chunks`` argument::
 
-This function will automatically concatenate and merge datasets into one in
-the simple cases that it understands (see :py:func:`~xarray.combine_by_coords`
-for the full disclaimer). By default, :py:func:`~xarray.open_mfdataset` will chunk each
-netCDF file into a single Dask array; again, supply the ``chunks`` argument to
-control the size of the resulting Dask arrays. In more complex cases, you can
-open each file individually using :py:func:`~xarray.open_dataset` and merge the result, as
-described in :ref:`combining data`. Passing the keyword argument ``parallel=True`` to
-:py:func:`~xarray.open_mfdataset` will speed up the reading of large multi-file datasets by
-executing those read tasks in parallel using ``dask.delayed``.
+        ds = xr.open_dataset("example-data.nc", chunks={"time": 10})
 
-.. warning::
+    Or open multiple files with py:func:`~xarray.open_mfdataset`::
 
-    :py:func:`~xarray.open_mfdataset` called without ``chunks`` argument will return
-    dask arrays with chunk sizes equal to the individual files. Re-chunking
-    the dataset after creation with ``ds.chunk()`` will lead to an ineffective use of
-    memory and is not recommended.
+        xr.open_mfdataset('my/files/*.nc', parallel=True)
 
-You'll notice that printing a dataset still shows a preview of array values,
-even if they are actually Dask arrays. We can do this quickly with Dask because
-we only need to compute the first few values (typically from the first block).
-To reveal the true nature of an array, print a DataArray:
+    .. tip::
 
-.. ipython:: python
+        When reading in many netCDF files with py:func:`~xarray.open_mfdataset`, using ``engine=h5netcdf`` can
+        be faster than the default which uses the netCDF4 package.
 
-    ds.temperature
+    Saving a larger-than-memory netCDF file::
 
-Once you've manipulated a Dask array, you can still write a dataset too big to
-fit into memory back to disk by using :py:meth:`~xarray.Dataset.to_netcdf` in the
-usual way.
+       delayed_write = ds.to_netcdf("my-big-file.nc", compute=False)
+       delayed_write.compute()
 
-.. ipython:: python
+    .. note::
 
-    ds.to_netcdf("manipulated-example-data.nc")
+        When using Dask’s distributed scheduler to write NETCDF4 files, it may be necessary to set the environment variable ``HDF5_USE_FILE_LOCKING=FALSE`` to avoid competing locks within the HDF5 SWMR file locking scheme. Note that writing netCDF files with Dask’s distributed scheduler is only supported for the netcdf4 backend.
```

Comment:
I didn't see anything in the release notes about a fix/update for this, so I'm assuming it's still true.

Reply:
Can you add a link to the GitHub issue to track this one?

Reply:
I couldn't find a corresponding open issue, but looks like Joe added it in https://github.com/pydata/xarray/pull/1793. 

I noticed `https://github.com/pydata/xarray/issues/1836`, is referenced, but I think this is a slightly different issue?